### ISSUE TITLE
Library artifacts are published to jcenter only, which will close in 3 months

### ISSUE THREAD

**Issue by entibaldi** [markormesher/android-fab#58]
Hello! Your library is published only to jcenter. Please consider publishing it also to mavenCentral, as jcenter is [shutting down in May](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/).

[Here](https://central.sonatype.org/pages/producers.html) is some info on how to publish artifacts to mavenCentral.

**Comment by france-workmail** [comment]
Anyone able to migrate from jcenter to mavenCentral with this library?

**Comment by markormesher** [comment]
I can take a look at doing that over the weekend or next week. Are you a consumer of this library?

**Comment by france-workmail** [comment]
Yes I am and I dont to migrate to anything else since we've been using this for the longest time.
